flow360/component/simulation/translator/solver_translator.py

```diff
@@ -74,6 +91,70 @@ def rotation_translator(model: Rotation):
     return volume_zone
 
 
+def porous_media_entity_info_serializer(volume):
+    """Porous media entity serializer"""
+    if isinstance(volume, Box):
+        axes = volume.private_attribute_input_cache.axes
+        return {
+            "zoneType": "box",
+            "axes": [list(axes[0]), list(axes[1])],
+            "center": list(volume.center),
+            "lengths": list(volume.size),
+        }
+
+    axes = volume.axes
+    return {
+        "zoneType": "mesh",
+        "zoneName": volume.name,
+        "axes": [list(axes[0]), list(axes[1])],
+    }
+
+
+def porous_media_translator(model: Rotation):
+    """Porous media translator"""
+    porous_medium = remove_units_in_dict(dump_dict(model))
+    porous_medium.pop("volumes")
+    porous_medium.pop("type")
+    porous_medium["DarcyCoefficient"] = list(porous_medium.pop("darcyCoefficient"))
+    porous_medium["ForchheimerCoefficient"] = list(porous_medium.pop("forchheimerCoefficient"))
+    return porous_medium
+
+
+def boundary_translator(model: SurfaceModelTypes, op_acousitc_to_static_pressure_ratio):
```

Comment:
can you rename it to `boundary_spec_translator`? `boundary_translator `sondes like it does everything including the entity part.